src/components/toast/toast.style.ts

```diff
@@ -180,11 +222,21 @@ const ToastStyle = styled(MessageStyle)<{
     `}
 `;
 
-const ToastContentStyle = styled(MessageContentStyle)<{
+const StyledToastContent = styled.div<{
   isNotice?: boolean;
   isDismiss?: boolean;
 }>`
   padding: 8px 16px 8px 16px;
+  white-space: pre-wrap;
+  flex: 1;
+
+  .carbon-content__title {
```

Comment:
Question: are these actually set anywhere? I think these have been missed when we converted [Content](https://github.com/Sage/carbon/blob/c94999f26e0bef0c72a6388afbb78c303100bdf1/src/components/content/content.component.tsx) to a styled component, the fact no one has complained leads me to think no one uses it or cares that it's not got the margin so maybe we just remove from both Toast and Message?

Reply:
When I did see it, I did think it was odd/legacy related. I'll remove it and see what breaks.

Reply:
Nothing broke. Guess it was just legacy left over code.